src/pages/stats.tsx

```diff
@@ -230,6 +236,21 @@ async function page(session: Session | null) {
             </span>
           )}
         </StatsCardHtml>
+        <StatsCardHtml title="Latest games">
```

Comment:
I recommend we add a 
`doubleSize?: boolean;` to the Props of `StatsCardHtml` then `({ title, children, doubleSize }: Props)`
```
<div
    class={cn(
      doubleSize ? "col-span-6 md:col-span-12" : "col-span-6",
      "flex h-full flex-col rounded-lg p-5 text-white",
    )}
  >
  ```
  
  This should make it take two columns on desktop and one on mobile

Reply:
black magic
added